site/templates/nav.jet

```diff
@@ -2,14 +2,14 @@
 {{import "./nav/user_logged_in.jet"}}
 
 {{block nav(showSubNav=true)}}
-<div class="navigation" data-toggle="affix">
-  <nav class="navbar navbar-expand-md navbar-dark">
+<header class="navigation" data-toggle="affix" role="banner">
```

Comment:
[Looks to be redundant](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Banner_role).

```suggestion
<header class="navigation" data-toggle="affix">
```